File: python-web-scraping/scrape.py
# How to scrape a webpage in python
import requests
from bs4 import BeautifulSoup

# Send a GET request to the webpage you want to scrape
response = requests.get("https://abdoulaye.pro")

# Parse the HTML content using Beautiful Soup
soup = BeautifulSoup(response.content, 'html.parser')

# Find the element(s) you want to scrape using Beautiful Soup's selectors
title = soup.select_one('title').text
heading = soup.select_one('h1').text
paragraphs = [p.text for p in soup.select('p')]

# Print the scraped data
print("Title: ", title)
print("Heading: ", heading)
print("Paragraphs: ", paragraphs)

File: 3/scrape.js
//How to scrape a webpage in node.js

const request = require('request');
const cheerio = require('cheerio');

// Send a GET request to the webpage you want to scrape
request('https://abdoulaye.pro', (error, response, html) => {
  if (!error && response.statusCode == 200) {
    // Load the HTML content into Cheerio
    const $ = cheerio.load(html);

    // Find the element(s) you want to scrape using Cheerio selectors
    const title = $('title').text();
    const heading = $('h1').text();
    const paragraphs = $('p').map((i, el) => $(el).text()).get();

    // Print the scraped data
    console.log('Title:', title);
    console.log('Heading:', heading);
    console.log('Paragraphs:', paragraphs);
  }
});